Header header
jsk_recognition_msgs/Histogram[] feature_list
int64[] labels